code/game/objects/items/reagent_containers/glass.dm

```diff
@@ -230,6 +231,72 @@
 		var/image/lid = image(icon, src, "lid_[initial(icon_state)]")
 		overlays += lid
 
+/obj/item/reagent_container/glass/minitank
+	name = "MS-11 Smart Refill Tank"
+	desc = "A robust little tank capable of refilling autoinjectors that previously required a nanomed system to refill. Using the wonders of microchips, it automatically sorts the correct chemicals into most single reagent autoinjectors. It is unable to partially fill them however. A valve exists on the top to transfer reagents to another container or to flush it entirely."
+	icon = 'icons/obj/items/tank.dmi'
+	icon_state = "mini_reagent_tank"
+	matter = list("metal" = 500)
+	attack_speed = 4
+	volume = 180
+	w_class = SIZE_MEDIUM
+	splashable = FALSE
```

Comment:
why dose this have to be a glass subtype? It's clearly not glass

Reply:
To inherit transferring, labeling and examining from glass. The pressurized reagent canister pouch is also glass too.
Now that you mention it there are a few oversights in the code, notably harm splashing not checking if its splashable and the container still being able to put in OT rockets among other things. Will be fixed.